### PR TITLE
Histogram upgrade

### PR DESCRIPTION
**Pull request by colinluthier**
@jekhokie I didn't make the new file executable but I think the rest is in place.

### CODE REVIEW

File: scripts/image_processors/noaa_pristine.sh [hunk 1]
```diff
@@ -18,4 +18,4 @@ INPUT_WAV=$1
 OUTPUT_IMAGE=$2
 
 # produce the output image
-$WXTOIMG -o -e "pristine" "${INPUT_WAV}" "${OUTPUT_IMAGE}"
+$WXTOIMG -o -e -16 "pristine" "${INPUT_WAV}" "${OUTPUT_IMAGE}"
```
Comment (Cadair) [inline scripts/image_processors/noaa_pristine.sh:21]: I am curious, what's this do?

File: scripts/prune_scripts/prune_older_than.sh [hunk 2]
```diff
@@ -12,8 +12,8 @@ let prunedate=$(date +%s)-$PRUNE_OLDER_THAN*24*60*60
 
 log "Pruning captures..." "INFO"
 for img_path in $(sqlite3 ${DB_FILE} "select file_path from decoded_passes where pass_start < $prunedate;"); do
-  find "${IMAGE_OUTPUT}/" -maxdepth 1 -type f -name "${img_path}*.jpg" exec rm -f {} \;
-  find "${IMAGE_OUTPUT}/thumb/" -maxdepth 1 -type f -name "${img_path}*.jpg" exec rm -f {} \;
+  find "${IMAGE_OUTPUT}/" -maxdepth 1 -type f -name "${img_path}*.jpg" exec rm -f {};
+  find "${IMAGE_OUTPUT}/thumb/" -maxdepth 1 -type f -name "${img_path}*.jpg" exec rm -f {};
```
Comment (Cadair) [inline scripts/prune_scripts/prune_older_than.sh:16]: I just merged this in the other PR, so this probably needs an update.

**Review by Cadair — COMMENTED** [review]
Seems sane, though I should probably run it.

You need to fix the fact `Vera.ttf` has gone walkies again.

File: scripts/image_processors/noaa_pristine.sh (hunk 1 above)
Comment (dom-robinson) [reply scripts/image_processors/noaa_pristine.sh:21]: The ```-16``` (from the manual): 

_Write 16 bit grayscale. Only PNG and PBM (PGM) formats can support 16 bit grayscale._

**Review by dom-robinson — APPROVED** [review]
From my point of view this checks out. @Cadair do you want to do the merge honours :)

**Comment by colinluthier** [comment]
The problem with this merge is the browser still looks for the jpg file.

**Comment by dom-robinson** [comment]
@colinluthier @Cadair are we 'go' with this? Want me to merge it in now?

**Comment by colinluthier** [comment]
I have not had time to look for the browser bug. It works on my production
system but I can’t merge from it and the changes code was added by Justin.
I have not found it.

**Comment by Cadair** [comment]
I think this PR needs master merging in?

**Comment by dom-robinson** [comment]
This looks like everyone was approving it but not actually merging it so i thought I’d ‘just do it’ :) 

Lets hope its not breaking :)

**Pull request merged**